website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -107,7 +107,7 @@ Hello world
 
 ## Specifying title {#specifying-title}
 
-You may also specify an optional title.
+You may also specify an optional title or omit content.
```

Comment:
I don't think we want to recommend this by default—it's a misuse of what admonitions are designed for. It's okay if you want to use it, but for the sake of testing let's just put these in `_dogfooding`.

Reply:
Moved to a separate file under `_dogfooding`.